src/pages/studyView/StudyViewPage.tsx

```diff
@@ -188,6 +189,9 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                                         <IFrameLoader height={700}
                                                       url={`//bioinformatics.mdanderson.org/TCGA/NGCHMPortal/?${this.store.MDACCHeatmapStudyMeta.result[0]}`}/>
```

Comment:
could we just the urls method to get the url for iframe?

Reply:
we should probably fix this in a separate PR